specification/compute/resource-manager/readme.typescript.md

```diff
@@ -0,0 +1,12 @@
+## TypeScript
+
+These settings apply only when `--typescript` is specified on the command line.
```

Comment:
Same as other comment. This file came in from Azure master actually to forked repository..